explainer.md

```diff
@@ -63,19 +63,22 @@ The basic steps most WebXR applications will go through are:
 
 The first thing that any XR-enabled page will want to do is request an `XRDevice` and, if one is available, advertise VR functionality to the user. (For example, by adding a button to the page that the user can click to start XR content.)
 
-`navigator.xr.requestDevice` returns a [`Promise`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise) that resolves to an `XRDevice` if one is available. If no `XRDevice` is available, it will resolve to null. The promise will be rejected if an error occurs, such as the page not having the appropriate permissions to access VR capabilities.
+`navigator.xr.requestDevice` returns a [`Promise`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise) that resolves to an `XRDevice` if one is available. If no `XRDevice` is available, it will reject with a `NotFoundError`. The promise will also be rejected if an error occurs, such as the page not having the appropriate permissions to access VR capabilities.
```

Comment:
are there any other notable [error names](https://heycam.github.io/webidl/#idl-DOMException-error-names) to call out here?

btw, for reference, the [Generic Sensor API](https://w3c.github.io/sensors/) throws a `SecurityException` if [permissions](https://w3c.github.io/sensors/#feature-detection) are not granted (and you didn't check using `navigator.permissions.query`).